grl/run.py

```diff
@@ -112,41 +112,41 @@ def add_tmaze_hyperparams(parser: argparse.ArgumentParser):
 
     # Run
     # Get POMDP definition
-    spec = load_spec(args.spec,
-                     memory_id=args.use_memory,
-                     n_mem_states=args.n_mem_states,
-                     corridor_length=args.tmaze_corridor_length,
-                     discount=args.tmaze_discount,
-                     junction_up_pi=args.tmaze_junction_up_pi,
-                     epsilon=args.epsilon,
-                     mem_leakiness=args.mem_leakiness)
+    amdp, pi_dict = load_pomdp(args.spec,
```

Comment:
thinking ahead to the POPGym integration: is there a good reason why we use `load_pomdp` rather than `get_env` here? I looked for usages of `pi_dict` and it seems this is only used when we have an `init_pi`; is this ever the case in practice?

This would be moot if we don't ever plan to use the main entrypoint with POPGym envs.

Reply:
Yes it's used for policy evaluation.